src/app/views/rolodex_view.js

```diff
@@ -0,0 +1,107 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+import $ from 'jquery';
+import Contact from '../models/contact.js';
+import ContactView from '../views/contact_view.js';
+import ContactDetailsView from '../views/contact_details_view.js';
+
+var RolodexView = Backbone.View.extend({
+  initialize: function(params){
+    this.template = params.template;
+
+    this.listenTo(this.model, "update", this.render);
+    $('.modal-container').hide();
```

Comment:
Should be `this.$('.modal-container').hide();`

Remember to avoid raw jQuery in a view.